Clarify installation and updating instructions

hCraker opened the pull request:
Closes #213 
- [x] add instructions on how to update geocat-comp
- [x] add instructions on how to install in pre-existing environment

erogluorhan reviewed (changes requested):
Nice work! Please also address this request below from GeoCAT repo [PR #43](https://github.com/NCAR/GeoCAT/issues/43) in this PR (Modify INSTALLATION.md to just link to docs/installation.rst):

- Keeping both INSTALLATION.md and docs/installation.rst files up-to-date and consistent with each other is a duplication of work. We can get one of these files just link to the other one (e.g. See Uxarray's [INSTALLATION.md](https://github.com/UXARRAY/uxarray/blob/main/INSTALLATION.md) that just links to UXarray docs installation page).

hCraker commented:
@erogluorhan I have addressed your suggestions in points 1 and 2. For point 3, I have been unable to find pip install instructions in any of our GeoCAT repositories. Do we have confirmed users who install via pip? If so, I can add the instructions for installing via PyPi to the installation.rst file, but I may need some guidance as I have never used pip before.

erogluorhan commented on INSTALLATION.md:
```diff
@@ -1,102 +1,7 @@
-see above):
-
-    pytest test
+For information about installing the entire GeoCAT project, see our
```
Maybe even remove this as this is already mentioned in [GeoCAT-comp Installation](https://geocat-comp.readthedocs.io/en/latest/installation.html)

hCraker replied:
I just removed it

pilotchute replied:
should this be marked resolved?

erogluorhan reviewed (commented):
This is great, thanks! I have a single inline comment as well as the following suggestion to address PyPI installation information:

You may consider to create a title "Installing GeocAT-comp via PyPI" just before the title "Building GeoCAT-comp from source", then you can write something like:

"GeoCAT-comp is distributed also in PyPI; therefore, the above Conda installation instructions should, in theory, apply to PyPI installation through using `pip install` commands instead of `conda install` wherever they occur."

Does it make sense?

erogluorhan reviewed (approved):
Great; thanks for doing this!
